src/applications/accredited-representative-portal/containers/POARequestsPage.jsx

```diff
@@ -1,17 +1,46 @@
 import React from 'react';
-
+import { useLoaderData, NavLink, Outlet } from 'react-router-dom';
 import DigitalSubmissionAlert from '../components/DigitalSubmissionAlert/DigitalSubmissionAlert';
-import POARequestsTableFetcher from '../components/POARequestsTableFetcher/POARequestsTableFetcher';
-import usePOARequests from '../hooks/usePOARequests';
 
 const POARequestsPage = () => {
+  const poaRequests = useLoaderData();
+
   return (
     <>
       <h1 data-testid="poa-requests-heading">Power of attorney requests</h1>
       <DigitalSubmissionAlert />
       <div className="poa-requests-page-table-container">
-        <h2 data-testid="poa-requests-table-heading">Requests</h2>
-        <POARequestsTableFetcher usePOARequests={usePOARequests} />
+        {poaRequests.length === 0 ? (
+          <p data-testid="poa-requests-table-fetcher-no-poa-requests">
+            No POA requests found
+          </p>
+        ) : (
+          <>
+            <div role="tablist" className="poa-request__tabs">
+              <NavLink
+                to="/poa-requests/pending"
+                className="poa-request__tab-link"
+                role="tab"
+                aria-selected={({ isActive }) => (isActive ? 'true' : 'false')}
+                aria-controls="panel-pending"
+                id="pending"
+              >
+                Pending requests
+              </NavLink>
+              <NavLink
+                to="/poa-requests/completed"
```

Comment:
These tabs contribute to the query that we use to search for POA requests. This query will be reflected in the query string parameters like `/poa-requests?status=completed`.

Reply:
see https://github.com/department-of-veterans-affairs/vets-website/pull/33602#discussion_r1889340810